### ISSUE TITLE
How to set to full screen

### ISSUE THREAD

**Issue by ByronFanglei** [latentcat/uvcanvas#11]
React+vite

code：
`<div className={"w-full h-full"}>
      <div className={"fixed -z-50 w-full h-full box-border"}>
        <Zenitho style={{ width: "100%", height: "100%" }} />
      </div>
</div>`

![8b8f0ab0-3032-4294-884c-14c0002bdb07](https://github.com/latentcat/uvcanvas/assets/45235628/86c8310c-13ef-4f2c-8b26-7e86fcb24ef6)

Thank you

**Comment by CPunisher** [comment]
This should be a bug in component `Zenitho`, as the other components don't seem to have this problem

**Comment by ByronFanglei** [comment]
Indeed, no problem using the other component's, hopefully it can be fixed

**Comment by ByronFanglei** [comment]
> This should be a bug in component `Zenitho`, as the other components don't seem to have this problem

Indeed, no problem using the other component's, hopefully it can be fixed

**Comment by Sidd5arth** [comment]
I was unable to reproduce this @ByronFanglei 

![image](https://github.com/user-attachments/assets/6dc8ebef-2f9e-4623-820a-96996480a66b)

You can see the height is more than 600px and its perfectly responsive.

https://github.com/user-attachments/assets/a8e90696-0ff6-4914-b341-03f29af29fe9
